src/pages/Home.js

```diff
@@ -20,38 +21,31 @@ import {
   Progress,
   Upload,
   message,
-  Button,
-  Timeline,
-  Radio,
-} from "antd";
-import {
-  ToTopOutlined,
-  MenuUnfoldOutlined,
-  RightOutlined,
-} from "@ant-design/icons";
-import Paragraph from "antd/lib/typography/Paragraph";
+} from 'antd';
 
-import Echart from "../components/chart/EChart";
-import LineChart from "../components/chart/LineChart";
+import Echart from '../components/chart/EChart';
+import LineChart from '../components/chart/LineChart';
 
-import ava1 from "../assets/images/logo-shopify.svg";
-import ava2 from "../assets/images/logo-atlassian.svg";
-import ava3 from "../assets/images/logo-slack.svg";
-import ava4 from "../assets/images/logo-spotify.svg";
-import ava5 from "../assets/images/logo-jira.svg";
-import ava6 from "../assets/images/logo-invision.svg";
-import team1 from "../assets/images/team-1.jpg";
-import team2 from "../assets/images/team-2.jpg";
-import team3 from "../assets/images/team-3.jpg";
-import team4 from "../assets/images/team-4.jpg";
-import card from "../assets/images/info-card-1.jpg";
+import ava1 from '../assets/images/logo-shopify.svg';
+import ava2 from '../assets/images/logo-atlassian.svg';
+import ava3 from '../assets/images/logo-slack.svg';
+import ava4 from '../assets/images/logo-spotify.svg';
+import ava5 from '../assets/images/logo-jira.svg';
+import ava6 from '../assets/images/logo-invision.svg';
+import team1 from '../assets/images/team-1.jpg';
+import team2 from '../assets/images/team-2.jpg';
+import team3 from '../assets/images/team-3.jpg';
+import team4 from '../assets/images/team-4.jpg';
 
 function Home() {
-  const { Title, Text } = Typography;
+  const { Title } = Typography;
 
-  const onChange = (e) => console.log(`radio checked:${e.target.value}`);
+  const [coins, setCoins] = useState();
+  const { handleAllCoins } = useApi();
 
-  const [reverse, setReverse] = useState(false);
+  // useEffect(() => {
+  //   handleAllCoins().then((res) => setCoins(res));
+  // }, []);
```

Comment:
ESLINT says to remove the dependency array from useEffect if it is empty (probably just a stylistic thing)

Reply:
If we remove the dependency array the API will be called for an infinite time, using an empty array [] we are saying to run the useEffect only one time every render. It is a common ESLINT error

Reply:
ah true!